R/DBConnection.R

```diff
@@ -362,6 +362,36 @@ setGeneric("dbListTables",
   valueClass = "character"
 )
 
+#' List remote objects
+#'
+#' Returns the names of remote tables accessible through this connection
+#' (possibly via a prefix) as a data frame.
+#' This should, where possible, include temporary objects.
```

Comment:
Maybe "This should include temporary objects, but not all database drivers support listing temporary objects"

Reply:
Perhaps include a sentence describing how this is different to `dbListTables()`?